docs/topics/native/native-spm.md

```diff
@@ -0,0 +1,192 @@
+[//]: # (title: Swift Package export setup)
+
+Kotlin/Native output for an Apple target can be consumed as an SPM dependency.
+For example, if your Kotlin Multiplatform project has an iOS target, you may want to make the binary available
+as a dependency to iOS developers working on native Swift projects in Xcode. Using KMP tooling, you can provide
```

Comment:
Here and below, please avoid the KMP acronym. In the docs, we try to use it as little as possible, as Kotlin has a long history of confusing multiplatform acronyms